app/players/elves/elf_ranger.py

```diff
@@ -0,0 +1,18 @@
+from app.players.elves.elf import Elf
+
+
+class ElfRanger(Elf):
+    def __init__(self, nickname: str,
+                 musical_instrument: str, bow_level: int) -> None:
+        super().__init__(nickname, musical_instrument)
+        self._bow_level = bow_level
+
+    def player_info(self) -> str:
+        return (f"Elf ranger {self.nickname}."
+                f" {self.nickname} has bow of the {self._bow_level} level")
+
+    def get_rating(self) -> int:
+        return self._bow_level * 3
+
+    def play_elf_song(self) -> None:
+        super().play_elf_song()

```

Comment:
The `play_elf_song` method calls `super().play_elf_song()`, which is an abstract method in the `Elf` class. Ensure that the `Elf` class's `play_elf_song` method is not abstract or has a proper implementation. Otherwise, this call will lead to a runtime error.